__tests__/testUtilities/stateUtils.js

```diff
@@ -229,6 +229,18 @@ const buildResourceWithLiteral = (state, options) => {
         authorities: [],
       },
   }
+
+  if (options.hasDefaultLiterals) {
+    state.entities.propertyTemplates[
+      "ld4p:RT:bf2:Title:AbbrTitle > http://id.loc.gov/ontologies/bibframe/mainTitle"
+    ].defaults = [
+      {
+        literal: "Default literal1",
+        lang: null,
+      },
+    ]
+  }
+
```

Comment:
update the tests to allow state to be created with a default value, so we can use in our new test above